new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful and popular editor that has many commands. Unlike most
+text editors where you use the keyboard to enter text and arrow keys to move you
+cursor, Vim uses the keyboard to enter commands to manipulate the text.
+There is a great amount of commands, too many to explain in a tutor such as
+this, but as you will see over the course of this tutorial series these commands
+are not arbitrarily chosen, they form a language with its own grammar.
+
+This tutor is designed to describe enough of the commands
+that you will be able to easily use Vim as an all-purpose editor.  It is
+IMPORTANT to remember that this tutor is set up to teach by use.  That means
+that you need to do the exercises to learn them properly.  If you only read
+the text, you will soon forget what is most important!
+
+NOTE: The commands in the lessons will modify the text, but those changes won't
+be saved.  Don't worry about messing things up; just remember that pressing
+[«<Esc>»](<Esc>) and then [u](u) will undo the latest change.
+
+This tutorial is interactive, and there are a few things you should know.
+Pressing [«<Enter>»](<Enter>) over text highlighted [like this](holy-grail) will take you to some relevant
+help (hopefully), and pressing ? over any word will try to do so too.  You can close
+the help window by pressing «q».  Sometimes you will be required to modify text like
+--->  this here {expect:this here}
+Once you have done the changes correctly, the ✗ sign at the left will change
+to ✓. I imagine you can already see how neat Vim can be ;)
+Other times, you'll be prompted to run a command (I'll explain this later):
+~~~ viml
+    :help <Enter>
+~~~
+or press a sequence of keys
+
+    <Esc>0f<SPACE>d3wP$P
+
+Text within <'s and >'s (like <Enter>) describes a key to press instead of text
+to type.
+
+Now, make sure that your Shift-Lock key is NOT depressed and press the «j»
+key enough times to move the cursor so that Lesson 1.1 completely fills the
+screen.
+
+## Lesson 1.1: MOVING THE CURSOR
+
+** To move the cursor, press the «h», «j», «k», «l» keys as indicated. **
+
+           ↑
+           k       Hint: The h key is at the left and moves left.
+        ←h   l→          The l key is at the right and moves right.
```

Comment:
Yeah, that is too elaborate. 

I think the analogy with the arrow keys is appropriate to the level of user the tutorial is aimed at, which might be used to move the cursor using the arrow keys. 

Besides, it is unclear in what sense the j and k "move things" up and down. At this point in the tutorial, the user only knows how to move the cursor, using motions in commands is yet ahead...